android/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,4 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.amazonivsreactnative">
+          package="com.amazonivsreactnativeplayer">
```

Comment:
To ensure correct namespacing across future products we should update this and related package identifiers to adhere to a consistent structure.

i.e. given changes from the framework PR #31, perhaps:

```
com.amazonaws.ivs.reactnative.player
```

Or similar. Will confirm internally if there are preferences.

Reply:
@arekkubaczkowski I've re-targeted this feature branch to a feature branch internally within the repo. It adds the necessary Android package rename changes. So once rebased we can merge to that branch first and re-test as needed.